tools/topology/sof-icl-rt711-rt1308-rt715-hdmi.m4

```diff
@@ -71,7 +73,7 @@ PIPELINE_PCM_ADD(sof/pipe-volume-playback.m4,
 
 # Low Latency capture pipeline 5 on PCM 4 using max 2 channels of s32le.
 # Schedule 48 frames per 1000us deadline on core 0 with priority 0
-PIPELINE_PCM_ADD(sof/pipe-highpass-capture.m4,
+PIPELINE_PCM_ADD(sof/pipe-eq-capture.m4,
```

Comment:
@singalsu can you comment here. @libinyang why is this change needed ? it does not appear in commit message.

Reply:
I don't know why this change is done but pipe-eq-capture adds an IIR high-pass with +20 dB gain and pipe-highpass-capture adds an IIR with 0 dB gain. It's possible to get the same gain by setting volume control to +20 dB with the latter topology. But since volume defaults to 0 dB and apparently we don't have a mechanism to pass a default gain from topology to driver so using the topology with gain offset is justified in some cases (e.g. need out of box Ubuntu to sound sufficiently loud). @juimonen and @libinyang know would be the experts to comment if some UCM setting could achieve the same.

Reply:
@lgirdwood @singalsu pipe-eq-capture.m4 creates the "DMIC Capture Switch" control. 
In Pipe-eq-catpure.m4, it defines:
```
C_CONTROLMIXER(Master Capture Switch, PIPELINE_ID,
        CONTROLMIXER_OPS(volsw, 259 binds the mixer control to switch get/put handlers, 259, 259),
        CONTROLMIXER_MAX(max 1 indicates switch type control, 1),
        false,
        ,
        Channel register and shift for Front Left/Right,
        LIST(`  ', KCONTROL_CHANNEL(FL, 2, 0), KCONTROL_CHANNEL(FR, 2, 1)),
        "1", "1")
```
This means the switch controller will enable MIC MUTE LED feature.
This is why we need pipe-eq-capture.m4

Reply:
ok, so why are we adding +20dB gain ? Should this not be set my UCM. Default should always be 0dB.

Reply:
@libinyang this is the wrong direction, sorry. There should be a capture switch for the rt715 that can be used. We should not overload the DMIC capture switch when we don't use the DMIC, and as a result end-up with an unnecessary 20dB boost.

Reply:
@lgirdwood @plbossart I was thinking that MIC is a mic like a DMIC. Thanks for the information. I add a new pipeline model: pipe-volume-switch-capture pipeline in my patch. Do you think it is OK to use this pipeline for the MIC? Or should I create a new pipeline pipe-highpass-volume-capture-capture for this MIC?

Reply:
@singalsu any comments on pipeline for this use case ?

Reply:
My point what that we should have a 'MIC Switch" control exposed by the rt715 driver. I don't see why we would add a switch on an SOF pipeline when the capture is controlled by an external device.

It's similar to HDAudio capture, we don't have a MIC switch for SOF, so why should we just when we change from HDaudio to SoundWire?

Reply:
@plbossart Yes, when I enabled this feature, I referred thinkpad laptop which uses sof HDaudio. To be honest, this confuses me. MIC MUTE LED is a machine related feature and codec should never realize this feature. I mean if we changed the codec to any other vendor's codec, the LED feature should always be there, because the led key press event is from ACPI level, not from codec level. So I prefer to implement this feature in the machine driver. And because our tplg is bound to machine tightly, I implement this feature in our tplg.